### PR TITLE
Remove custom appender include plugin

### PR DESCRIPTION
**Pull request by Calum-Campbell**
Tested on CODE

### CODE REVIEW

File: cloudformation/identity-admin-api.yaml [hunk 1]
```diff
@@ -326,7 +330,9 @@ Resources:
                 file {
                     add_field => {
                         'application' => '${App}'
```
Comment (jfsoul) [inline cloudformation/identity-admin-api.yaml]: I think just throw this away as we are using `app` now

File: cloudformation/identity-admin-api.yaml [hunk 2]
```diff
@@ -177,7 +180,8 @@ Resources:
             Action:
             - kinesis:Describe*
             - kinesis:Put*
-            Resource: '*'
+            Resource:
+            - arn:aws:kinesis:eu-west-1:942464564246:stream/identity-logging-PROD-KinesisStream-G0Y2BTGKK20H
```
Comment (jfsoul) [inline cloudformation/identity-admin-api.yaml]: Should this not use the parameter?

Comment (Calum-Campbell) [reply cloudformation/identity-admin-api.yaml]: I could make a new parameter for it, this is arn as opposed to the kinesis stream name, that we already have a param for. In other projects we've actually just set it to : 
```
Resource: '*'
```

File: cloudformation/identity-admin-api.yaml (hunk 1 above)
Comment (Calum-Campbell) [reply cloudformation/identity-admin-api.yaml]: ok will do

File: cloudformation/identity-admin-api.yaml (hunk 2 above)
Comment (jfsoul) [reply cloudformation/identity-admin-api.yaml]: Hmm ok - could we do something like `!Sub arn:aws:kinesis:eu-west-1:$(AwsAccount}:stream/${KinesisStream}`?

Comment (Calum-Campbell) [reply cloudformation/identity-admin-api.yaml]: Done

**Pull request merged**